tests/target-browser/target_browser_test.go

```diff
@@ -927,48 +939,59 @@ var _ = Describe("Target Browser Tests", func() {
+			It(fmt.Sprintf("Should get both OperationScoped backup and clusterbackups if flag %s is not provided", flagOperationScope), func() {
 				args := []string{cmdGet, cmdBackup}
-				args = append(args, commonArgs...)
-				cmd := exec.Command(targetBrowserBinaryFilePath, args...)
-				_, err := cmd.CombinedOutput()
-				Expect(err).ShouldNot(HaveOccurred())
+				backupData := runCmdBackup(args)
+				Expect(len(backupData)).To(Equal(noOfBackupPlansToCreate * noOfBackupsToCreate * 2))
 			})
 
-			It(fmt.Sprintf("Should succeed if flag %s is given zero value", flagOperationScope), func() {
-				args := []string{cmdGet, cmdBackup, flagOperationScope, ""}
-				args = append(args, commonArgs...)
-				cmd := exec.Command(targetBrowserBinaryFilePath, args...)
-				_, err := cmd.CombinedOutput()
-				Expect(err).ShouldNot(HaveOccurred())
+			It(fmt.Sprintf("Should get both backups and clusterbackups if flag %s is given zero value", flagOperationScope), func() {
+				args := []string{cmdGet, cmdBackup}
+				backupData := runCmdBackup(args)
+				Expect(len(backupData)).To(Equal(noOfBackupPlansToCreate * noOfBackupsToCreate * 2))
 			})
 
-			It(fmt.Sprintf("Should get %d backupPlan if flag %s is given %s value", noOfBackupPlansToCreate,
+			It(fmt.Sprintf("Should get %d backup if flag %s is given %s value", noOfBackupPlansToCreate,
 				flagOperationScope, internal.MultiNamespace), func() {
 				args := []string{cmdGet, cmdBackup, flagOperationScope, internal.MultiNamespace}
 				backupData := runCmdBackup(args)
 				Expect(len(backupData)).To(Equal(noOfBackupPlansToCreate))
+				for idx := range backupData {
+					Expect(backupData[idx].Kind).To(Equal(internal.ClusterBackupKind))
+				}
 			})
 
 			It(fmt.Sprintf("Should get 0 backupPlan if flag %s is given %s value",
 				flagOperationScope, internal.SingleNamespace), func() {
 				args := []string{cmdGet, cmdBackup, flagOperationScope, internal.SingleNamespace}
 				backupData := runCmdBackup(args)
-				Expect(len(backupData)).To(Equal(0))
+				for idx := range backupData {
+					Expect(backupData[idx].Kind).To(Equal(internal.BackupKind))
+				}
 			})
 
-			It(fmt.Sprintf("Should get one backup for specific cluster backup UID %s", backupUID), func() {
-				isLast = true
-				args := []string{cmdGet, cmdBackup, backupUID, flagOperationScope, internal.MultiNamespace}
+			It(fmt.Sprintf("Should get one backup for specific backup UID %s if flag %s is given %s value",
+				backupUID, cmd.OperationScopeFlag, internal.SingleNamespace), func() {
+				args := []string{cmdGet, cmdBackup, backupUID, flagOperationScope, internal.SingleNamespace}
 				backupData := runCmdBackup(args)
 				Expect(len(backupData)).To(Equal(1))
 				Expect(backupData[0].UID).To(Equal(backupUID))
+				Expect(backupData[0].Kind).To(Equal(internal.BackupKind))
 			})
 
+			It(fmt.Sprintf("Should get one backup for specific cluster backup UID %s if flag %s is given %s value",
```

Comment:
add similar test case with OperationScopeFlag=SingleNamespace without OperationScopeFlag

do the same for backupPlan cmd tests you've added